releasenotes/notes/0.25/add-pauli-equivalences-74c635ec5c23ee33.yaml

```diff
@@ -1,7 +1,9 @@
 ---
-features:
+features_transpiler:
   - |
-    The transpiler's built-in :class:`.EquivalenceLibrary` has been taught the circular Pauli
+    The transpiler's built-in :class:`.EquivalenceLibrary`
+    (singleton ``qiskit.circuit.equivalence_library.SessionEquivalenceLibrary``)
+    has been taught the circular Pauli
```

Comment:
`SessionEquivalenceLibrary` isn't technically a singleton instance - there's also `StandardEquivalenceLibrary` which is the Qiskit-internal base that we use, and that's what actually gained the new equivalences.  The `Session` one inherits rules from the base, and that's the one we (sort of) expose as mutable to users, and it's technically possible to run the `BasisTranslator` with a totally manually constructed set of equivalences.

That said, I'm not _super_ sure that there's a reason we need to make that split haha.